### PR TITLE
Add vscode urls to the review page

### LINKED ISSUE #21403: Assay urls on the review page
We want `vscode://mozilla.assay/review/guid/version` urls in the review page to use the new code review tool

Screenshot 2023-11-06 at 12 53 02 PM

This is a mockup of such a thing

### PR DESCRIPTION
**Pull request by GregSherman**
Fixes #21403 

Screenshot 2023-11-07 at 7 45 43 PM

Adds in this link that opens vscode to download the addon.

### CODE REVIEW

**Comment by GregSherman** [comment]
@eviljeff requesting review

File: src/olympia/reviewers/templates/reviewers/includes/files_view.html [hunk 1]
```diff
@@ -4,6 +4,14 @@
   &middot;
   <a href="{{ url('devhub.file_validation', addon.slug, file.id) }}">Validation results</a>
   &middot;
+  <a 
+    class="assay"
+    href="vscode://mozilla.assay/review/{{addon.guid}}/{{version}}"
+    title="Download this version in the VS Code addon reviewer"
```
Comment (eviljeff) [inline src/olympia/reviewers/templates/reviewers/includes/files_view.html:10]: nit, but this title reads a bit strange to me.

**Review by eviljeff — APPROVED** [review]

**Comment by GregSherman** [comment]
@eviljeff can we get this merged? (no write access)

**Comment by eviljeff** [comment]
> @eviljeff can we get this merged? (no write access)

sure. I thought you did for addons-server.

**Pull request merged**